scripts/userstyles.yml

```
@@ -914,4 +917,11 @@ userstyles:
     readme:
       app-link: "https://nextjs.org"
     current-maintainers: [*Dandraghas]
+  duolingo:
+  name: Duolingo
+  categories: [education]
+  color: green
+  readme:
+    app-link: "https://duolingo.com"
+  current-maintainers: [*Krymancer]
```

This needs to be indented properly, and please place this entry alphabetically instead of at the bottom.
```suggestion
  duolingo:
    name: Duolingo
    categories: [education]
    color: green
    readme:
      app-link: "https://duolingo.com"
    current-maintainers: [*Krymancer]
```